config/state-machine.php

```diff
@@ -50,6 +50,18 @@
                 'from' => ['printed','dispatched','allocated'],
                 'to' =>  'recorded',
             ],
+            'reject-to-printed' => [
```

Comment:
So complicated! :)

Reply:
I know. :cry: 

I thought "revert", but it isn't reverting, really. that implies we rollback the whole transaction and remove it, whereas i think we need a record of all transactions.